src/AdoPat/PatClient.cs

```diff
@@ -0,0 +1,143 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+namespace Microsoft.Authentication.AdoPat
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Threading;
+    using System.Threading.Tasks;
+    using Microsoft.VisualStudio.Services.DelegatedAuthorization;
+
+    /// <summary>
+    /// An abstraction over common operations with the Azure DevOps PAT Lifecycle Management REST API.
+    /// </summary>
+    public class PatClient
+    {
+        private const int PageSize = 100; // Using the maximum allowable page size allows us to reduce HTTP calls.

```

Comment:
This might be part of another PR, but we should consider moving all constants to `Constants.cs` for consistentancy.

Reply:
`Constant.cs` is part of the MSALWrapper, which is another namespace. I'd prefer not to mix namespaces like that. We could move the constants into a shared namespace, but I'd also avoid doing that because it reduces the locality of information. That is, you have to go further from the point of use to find out what a value is. This constant isn't used anywhere outside of this file (currently), so I'd prefer to leave it where it is for the foreseeable future.